dev-docs/SPEC.md

```diff
@@ -101,3 +113,32 @@ sequenceDiagram
 ```tlb
 claim_ton#0393b1ce receiver:MsgAddress = InternalMsgBody;
 ```
+
+## Semantic changes
+
+We decided to change behavior of a few receivers
+
+#### Mint send amount
+
+In other implementations, `mint` TL-B looks like this:
+
+```tlb
+mint#21
+  query_id:uint64
+  to_address:MsgAddressInt
+  ton_amount:Coins
+  master_msg:^JettonInternalTransfer
+  = JettonMinterMsg;
+```
+
+Field `ton_amount` represents amount that we want to send with out `master_msg` that is `JettonInternalTransfer`. However, since **atomic** receiver and it's only functionality is to perform checks and send next message in the chain, it make sense for it to follow general semantics in such cases, which is
```

Comment:
```suggestion
The `ton_amount` field represents the amount that we want to send without `master_msg`, which is `JettonInternalTransfer`. However, since it's an **atomic** receiver and its only functionality is to perform checks and send the next message in the chain, it makes sense for it to follow general semantics in such cases, which is
```